milestone2.ts

```diff
@@ -83,7 +83,13 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage = (data: Volunteer[]): number => {
```

Comment:
We provided the GetNumber type for this function already so you could have used that instead of redefining all of the types.